<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StockDataDialog</class>
 <widget class="QDialog" name="StockDataDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить позицию хранения</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="grpProduct">
       <property name="title">
        <string>Позиция хранения</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="editSearchProduct">
          <property name="placeholderText">
           <string>Поиск номенклатуры...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="listProduct"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="lblLocation">
           <property name="text">
            <string>Место хранения:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboLocation"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblProject">
           <property name="text">
            <string>Тема:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="comboProject"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lblProductName">
           <property name="text">
            <string>Выбранный товар:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="editProductName">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="editProductId">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblProductId">
           <property name="text">
            <string>Код товара:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lblAmount">
           <property name="text">
            <string>Количество:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="spinAmount">
           <property name="specialValueText">
            <string/>
           </property>
           <property name="maximum">
            <number>10000000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnOk">
           <property name="text">
            <string>Записать</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCancel">
           <property name="text">
            <string>Отменить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>StockDataDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
